build.gradle

```diff
@@ -441,31 +440,49 @@ localization.script = 'scripts/syncLang.py'
 // Test tasks
 test {
     useJUnitPlatform {
-        excludeTags 'DatabaseTest', 'FetcherTest', 'GUITest', 'org.jabref.testutils.category.FetcherTest', 'org.jabref.testutils.category.GUITest'
+        excludeTags 'DatabaseTest', 'FetcherTest', 'GUITest'
     }
 
     testLogging {
         // set options for log level LIFECYCLE
-        events "failed"
+        events = ["FAILED", "STANDARD_OUT", "STANDARD_ERROR"]
```

Comment:
@koppor just change this lines. 
Im only on mobile for the next days without access to a PC 💻